### PR TITLE
Fix# 17: Created XML Layout of Admin Dashboard

### PR DESCRIPTION
**Pull request by aniket0742**
## Overview

1. This PR fixes or fixes part of #10 .
2. This PR does the following: Created the xml layout for the Admin Dashboard with separate xml layout for card.

## Essential Checklist

- [☑️] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [☑️] "Allow edits from maintainers" is checked. (See [here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) for instructions on how to enable it.)
- [☑️] The PR is made from a branch that's **not** called "main/master".

## Proof that changes are correct
Screenshot 2024-06-16 033732

Screenshot 2024-06-16 032056

## PR Pointers

- If you need a review or an answer to a question, and don't have permissions to assign people, **leave a comment** like the following: "{{Question/comment}} @{{reviewer_username}} PTAL".
- Never force push. If you do, your PR will be closed.

### CODE REVIEW

File: app/src/main/res/layout/activity_card_entry_exit.xml
Comment (aarav0012) [inline app/src/main/res/layout/activity_card_entry_exit.xml:1]: show me the layout picture of card view

File: app/src/main/res/layout/activity_admin_dash_board.xml [hunk 1]
```diff
@@ -7,6 +7,111 @@
...
+        android:background="@color/blue"
+        app:layout_constraintTop_toBottomOf="@+id/title_dashboard"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        android:layout_marginTop="16dp"
+        android:layout_marginEnd="16dp"
+        android:layout_marginStart="16dp"/>
+    <Button
+        android:id="@+id/btnAvatar"
+        android:layout_width="36dp"
```
Comment (aarav0012) [inline app/src/main/res/layout/activity_admin_dash_board.xml]: height and width of avatar should be minimum 48 dp

**Comment by aniket0742** [comment]
changed the avatar dimensions.

**Comment by aniket0742** [comment]
layout picture for the card view 
Screenshot 2024-06-18 215018

**Comment by aniket0742** [comment]
Layout picture for the card view.

Screenshot 2024-06-18 215629

**Pull request merged**